test/common-tests/request-field-validation-api-tests/partials/require-validation.ts

```diff
@@ -0,0 +1,93 @@
+import { prepareModifiedRequest } from '@ukef-test/support/helpers/request-field-validation-helper';
+import request from 'supertest';
+
+export interface RequiredFieldValidationApiTestOptions<RequestBodyItem, RequestBodyItemKey extends keyof RequestBodyItem> {
+  fieldName: RequestBodyItemKey;
+  required?: boolean;
+  pattern?: RegExp;
+  enum?: any;
+  typeNameForErrorMessages: string;
+  generateFieldValueOfLength?: (length: number) => RequestBodyItem[RequestBodyItemKey];
+  generateFieldValueThatDoesNotMatchEnum?: () => RequestBodyItem[RequestBodyItemKey];
+  validRequestBody: RequestBodyItem[] | RequestBodyItem;
+  makeRequest: ((body: unknown[]) => request.Test) | ((body: unknown) => request.Test);
+  givenAnyRequestBodyWouldSucceed: () => void;
+}
+
+export function withRequiredFieldValidationApiTests<RequestBodyItem, RequestBodyItemKey extends keyof RequestBodyItem>({
+  fieldName: fieldNameSymbol,
+  required,
+  enum: theEnum,
+  typeNameForErrorMessages,
+  generateFieldValueThatDoesNotMatchEnum,
+  validRequestBody,
+  makeRequest,
+  givenAnyRequestBodyWouldSucceed,
+}: RequiredFieldValidationApiTestOptions<RequestBodyItem, RequestBodyItemKey>): void {
+  const fieldName = fieldNameSymbol.toString();
+  const requestIsAnArray = Array.isArray(validRequestBody);
+  const requestBodyItem = requestIsAnArray ? validRequestBody[0] : validRequestBody;
+
+  required = required ?? true;
```

Comment:
`required = Boolean(required)`?

Reply:
Typescript ensure that `required` will be boolean and not string like 'true', so this transformation is not required.